atsign-foundation/at_libraries
# Move IoT Safe code examples to at_chops

**- What I did**
- Moved IoT safe code from at_talk repo to at_chops
**- How I did it**
- packages/at_chops/example/zariot/compute_verify_signature_sim.dart
 This example will generate a new key pair, compute signature using generated ECC private keyfor a pre defined hash (hello world) and verify the signature using ECC public key. 
 - packages/at_chops/example/zariot/external_signer.dart
 All AT commands to interact with IoT Safe interface are implemented in external_signer
-  packages/at_chops/example/zariot/at_chops_secure_element.dart
implementation of at_chops which delegates signing functionality to external_signer
- packages/at_chops/example/zariot/onboard_secure_element.dart
sample code to test onboarding for an atsign whose pkam private key is in the sim card.
**- How to verify it**
- Run compute_verify_signature_sim.dart in a pi device which has sim mounted. Verify result should be success
- Run onboard_secure_element.dart in a pi device 
pre conditions:  atsign is provisioned in staging/prod. Atsign is activated through registrar API and cram secret should be available.
**- Description for the changelog**
